src/mixins/nr-affiliation-mixin.ts

```diff
@@ -231,10 +234,11 @@ export class NrAffiliationMixin extends Mixins(CommonMixin) {
       } else if (this.isContinuationIn) {
         this.setContinuationInErrorStatus(true)
         throw new Error('Unable to Continue In Now ' + error)
-      } else {
+      } else if (this.isNewBusiness) {
         this.setIncorporateNowErrorStatus(true)
         throw new Error('Unable to Incorporate Now ' + error)
       }
+      throw new Error('Unable to do action ' + error)
```

Comment:
I would change the text to "Error in actionNumberedEntity() = " to make it easier to trouble-shoot later if this case ever happens.

Reply:
Also, please add a `=` or `-` between the text and the error on lines 233, 236, 239 and 241.